tests/nnstreamer_source/unittest_src_iio.cc

```diff
@@ -1574,17 +1520,20 @@ TEST (test_tensor_src_iio, data_verify_freq_generic_type)
   clean_iio_dev_structure (dev0);
 }
 
-/**
- * @brief test the unusual/exceptional cases.
- */
-/** @todo Ignore src iio unusual_cases on ubuntu. 
+/** @todo Ignore src iio unusual_cases on ubuntu.
```

Comment:
`@brief` has been removed.

Reply:
They are moved to each function definition line of #if/#else

Reply:
got it